src/current/molt/molt-fetch.md

```diff
@@ -369,10 +373,110 @@ To cancel replication, enter `ctrl-c` to issue a `SIGTERM` signal. This returns
+{{site.data.alerts.end}}
+
+**Use a secure changefeed connection whenever possible.** The [default insecure configuration](#default-insecure-changefeed) is **not** recommended in production. To secure the changefeed connection, define `sink_query_parameters` in the JSON as follows:
+
+{% include_cached copy-clipboard.html %}
+~~~ json
+{
+    "sink_query_parameters": "client_cert={base64 cert}&client_key={base64 key}&ca_cert={base64 CA cert}"
+}
+~~~
+
+`client_cert`, `client_key`, and `ca_cert` are [webhook sink parameters]({% link {{ site.current_cloud_version }}/changefeed-sinks.md %}#webhook-parameters) that must be base64- and URL-encoded (for example, `base64 -i ./client.crt | jq -R -r '@uri'`).
+
+In the `molt fetch` command, also include [`--replicator-flags`](#global-flags) to specify the paths to the server certificate and key that correspond to the client certs defined in `sink_query_parameters`. For example:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+--changefeeds-path 'changefeed-secure.json'
+--replicator-flags "--tlsCertificate ./certs/server.crt --tlsPrivateKey ./certs/server.key"
+~~~
+
+For a complete example of using `molt fetch` in `failback` mode, see [Fail back securely from CockroachDB](#fail-back-securely-from-cockroachdb).
+
+##### Default insecure changefeed
+
+{{site.data.alerts.callout_danger}}
+Insecure configurations are **not** recommended. In production, run failback with a secure changefeed connection. For details, see [Changefeed override settings](#changefeed-override-settings).
+{{site.data.alerts.end}}
+
+When `molt fetch --mode failback` is run without specifying `--changefeeds-path`, the following [`CREATE CHANGEFEED` parameters]({% link {{ site.current_cloud_version }}/create-changefeed.md %}#parameters) are used for the changefeed:
+
+~~~ json
+{
+    "host": "localhost",
+    "port": 30004,
+    "sink_query_parameters": "insecure_tls_skip_verify=true",
+    "resolved": "1s",
+    "min_checkpoint_frequency": "1s",
+    "initial_scan": "no",
+    "webhook_sink_config": "{\"Flush\":{\"Bytes\":1048576}}"
+}
+~~~
+
+The default parameters specify a local webhook sink (`"localhost"`) and an insecure sink connection (`"insecure_tls_skip_verify=true"`), which are suited for testing only. In order to run `failback` with the default insecure configuration, you must also include the following flags:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+--allow-tls-mode-disable
+--replicator-flags '--tlsSelfSigned --disableAuthentication'
+~~~
+
+{{site.data.alerts.callout_info}}
+If both `--changefeeds-path` and `--allow-tls-mode-disable` are not specified in `failback` mode, `molt fetch` will error. At least one of these flags is required. `--changefeeds-path` overrides the default changefeed configuration. `--allow-tls-mode-disable` enables the use of the default changefeed configuration, which is insecure.
```

Comment:
```suggestion
Either
- `--changefeeds-path` which overrides the default changefeed configuration, or
- `--allow-tls-mode-disable` which enables the use of the default insecure changefeed configuration
must be specified in `failback` mode, otherwise `molt fetch` will error.
```

Reply:
Adapted this to be clearer. I'm not a fan of placing bullet points in the middle of a sentence as this doesn't look entirely right to me. Instead, I wrote it into a full sentence without bullet points.